sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnnecessaryUsings.cs

```diff
@@ -88,6 +90,28 @@ namespace MyNamespace3
     class Ns3_0 { }
 }
 
+namespace ReferencesInsideDocumentationTags
+{
+    using System.ComponentModel;
+
+    /// <summary><see cref="Win32Exception"/></summary>
```

Comment:
Just update a test a little to demonstrate that it really iterates all of them in single statement as well:
```suggestion
    /// <summary>There's <see cref="Win32Exception"/> or <see cref="WarningException"/>.</summary>
```

Reply:
I added another class instead, as `WarningException` is from the same namespace as the other, so it would be compliant even if it stopped at the first found class.